extensions/ql-vscode/src/pure/interface-types.ts

```diff
@@ -370,14 +371,15 @@ export type FromRemoteQueriesMessage =
   | RemoteQueryErrorMessage;
 
 export type ToRemoteQueriesMessage =
-  | OpenRemoteQueriesViewMessage;
+  | SetRemoteQueryResultMessage;
 
 export interface RemoteQueryLoadedMessage {
   t: 'remoteQueryLoaded';
 }
 
-export interface OpenRemoteQueriesViewMessage {
-  t: 'openRemoteQueriesView';
+export interface SetRemoteQueryResultMessage {
+  t: 'setRemoteQueryResult';
+  d: RemoteQueryResult
```

Comment:
We've only been using single char property names for `t`.

```suggestion
  data: RemoteQueryResult
```

or maybe:

```suggestion
  result: RemoteQueryResult
```

Reply:
Ah thanks, updated!